mods/ra/maps/siberian-conflict-1-fresh-tracks/rules.yaml

```diff
@@ -0,0 +1,71 @@
+World:
+	LuaScript:
+		Scripts: fresh-tracks.lua
+	MissionData:
+		WinVideo: sovbatl.vqa
+		LossVideo: overrun.vqa
+		Briefing: Soviet activity in Siberia is increasing. A large shipment of atomic material is on its way through this area, and we need it stopped. Destroy all of the Soviet convoy trucks and prevent the Soviets from furthering their atomic testing.\n\nBe careful of the convoy truck cargo -- atomic material is highly unstable.\n\nThe mission is complete when all the convoy trucks from each of the seven convoys have been destroyed.
+	ScriptLobbyDropdown@difficulty:
+		ID: difficulty
+		Label: Difficulty
+		Values:
+			easy: Easy
+			normal: Normal
+			hard: Hard
+		Default: normal
+
+Player:
+	PlayerResources:
+		DefaultCash: 2100
+
+MONEYCRATE:
+	Crate:
+		Lifetime: 0
+	ExplodeCrateAction:
+		Weapon: Atomic
+	Tooltip:
+		Name: Atomic Material
+
+TRUK:
+	Buildable:
+		Prerequisites: ~disabled
+
+SPY:
+	Buildable:
+		Prerequisites: ~disabled
+
+MECH:
+	Buildable:
+		Prerequisites: ~disabled
+
+E7:
+	Buildable:
+		Prerequisites: ~disabled
+
+TRAN:
+	Buildable:
+		Prerequisites: ~disabled
+
+HELI:
+	Buildable:
+		Prerequisites: ~disabled
```

Comment:
Is this intentional?

Reply:
Yes, the starting helicopter was not replaceable in this mission.